packages/configure-splash-screen/src/constants.ts

```diff
@@ -27,3 +33,22 @@ export interface Parameters {
    */
   darkModeImagePath?: string;
 }
+
+/**
+ * These might be optionally provided by the user. There are default values for them.
+ */
+export interface StatusBarOptions {
+  statusBarHidden: boolean;
+  statusBarStyle: StatusBarStyle;
+}
+
+/**
+ * These might be optionally provided by the user. There are default values for them.
+ * Android only
+ */
+export interface AndroidOnlyStatusBarOptions {
```

Comment:
Maybe `Partial` type?

Reply:
Matter of discussion, but I prefer `interface` over `type` when possible 😛 Moreover, when it comes to original interface/type IMO it's better to go with explicit (via `?` modifier) than implicit via `Partial` (but matter of implicity/explicity of `?` and `Partial` is another topic for discussion)